song-server/src/main/java/bio/overture/song/server/service/ValidationService.java

```diff
@@ -62,23 +68,26 @@ public class ValidationService {
   private final UploadRepository uploadRepository;
   private final boolean enforceLatest;
   private final Schema analysisTypeIdSchema;
+  private final RestTemplate restTemplate;
 
   @Autowired
   public ValidationService(
       @Value("${schemas.enforceLatest}") boolean enforceLatest,
       @NonNull SchemaValidator validator,
       @NonNull AnalysisTypeService analysisTypeService,
       @NonNull Supplier<Schema> analysisTypeIdSchemaSupplier,
-      @NonNull UploadRepository uploadRepository) {
+      @NonNull UploadRepository uploadRepository,
+      @NonNull RestTemplate restTemplate) {
     this.validator = validator;
     this.analysisTypeService = analysisTypeService;
     this.uploadRepository = uploadRepository;
     this.enforceLatest = enforceLatest;
     this.analysisTypeIdSchema = analysisTypeIdSchemaSupplier.get();
+    this.restTemplate = restTemplate;
   }
 
   @SneakyThrows
-  public Optional<String> validate(@NonNull JsonNode payload) {
+  public Optional<String> validate(@NonNull JsonNode payload, @NonNull String studyId) {
```

Comment:
`studyId` variable is not used in this method.

Reply:
It's not used now, but this will be used when we invoke external URL method